src/views/Farms/components/Table/Actions/HarvestAction.tsx

```diff
@@ -8,9 +8,14 @@ import { useHarvest } from 'hooks/useHarvest'
 import useI18n from 'hooks/useI18n'
 import { usePriceCakeBusd } from 'state/hooks'
 import { useCountUp } from 'react-countup'
+import styled from 'styled-components'
 
 import { ActionContainer, ActionTitles, Title, Subtle, ActionContent, Earned, Staked } from './styles'
 
+const HarvestButton = styled(Button)`
```

Comment:
you don't need to create a new component for this.
Most of the uikit component use the [space API from styled-system](https://styled-system.com/api/#space)
which means you can use `<Button ml="4px">....</Button>` instead of this